src/guides/v2.3/ui_comp_guide/components/ui-datecolumn.md

```diff
@@ -10,6 +10,8 @@ The DateColumn component implements a table column that displays dates.
 | Option | Description | Type | Default |
 | --- | --- | --- | --- |
 | `dateFormat` | Date format for the displayed column's values. | String | `MMM d, YYYY h:mm:ss A` |
+| `skipTimeZoneConversion` | If time is not important for filtering. | Boolean | `true` |
```

Comment:
Hi @eduard13. We may improve the description slightly. As a developer I don't completely understand the 

> time is not important for filtering

statement. Are we talking only about time or a timezone? Please, provide a more expanded explanation for this option. 

Thank you!

Reply:
Hi @rogyar, the description was slightly updated, based on the implementation of this method, which looks to use the `skipTimeZoneConversion` flag 
https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/Component/Form/Element/DataType/Date.php#L117-L119

Thank you.

Reply:
That makes sense since the `setUtcTimezone` argument is being retrieved from the [configuration](https://github.com/magento/magento2/blob/f5a7ac98dadb79e5c994fee06a5e03b8346e1671/app/code/Magento/Ui/Component/Filters/Type/Date.php#L138)